src/chapters/setup-intro-data-types/data-types.rst

```diff
@@ -84,95 +81,80 @@ are a few examples of popular languages falling into these two categories:
 #. **Dynamic**: Python, Ruby, Javascript, PHP
 #. **Static**: Java, C, C++, C#, TypeScript
 
-This major difference between JavaScript and Java means that we need to pay
-much more attention to types when writing Java code. Let’s begin by exploring
-the most common data types in this language.
+Because we need to give plenty of attention to types when writing Java code, let’s begin 
+by exploring the most common data types in this language.
+
+Strings and Single Characters
+-----------------------------
+
+Immutability
+^^^^^^^^^^^^
 
-String
--------
+Strings in Java are immutable. This means a string cannot be changed; A given character 
```

Comment:
The second sentence is redundant.  The students should have experience with immutability, so we can probably shorten the sentence.  Something like, "Strings in Java are *immutable*, which means that the characters within a string cannot be changed."